app.js

```diff
@@ -1,65 +1,54 @@
-function Star(el, count, callback) {
-    const starDiv = document.querySelector('#star');
-    const displayStar = document.querySelector('#display-star');
-    const starElements = document.querySelectorAll('#star i');
-    starElements.forEach((ele, index) => {
-        ele.setAttribute('value', index);
-    })
-    const addMouseEnter = (e) => {
-        let value = Number(e.target.getAttribute('value'));
-        starElements.forEach((ele, index) => {
-            if (index <= value) {
-                ele.classList.remove('fa-star-o');
-                ele.classList.add('fa-star');
-            }
-
-        })
-    };
-    starElements.forEach((ele) => {
-        ele.addEventListener('mouseenter', addMouseEnter);
-    });
-    const normaliseStars = (e) => {
-        starElements.forEach(ele => {
-            ele.classList.remove('fa-star');
-            ele.classList.add('fa-star-o');
-        })
+class Star {
+    constructor(el, count, callback) {
```

Comment:
Trying passing one Options or GlobalProp object instead of separate parameters while creating constructor.